### ISSUE TITLE
Full UI: Review mode - Skill Review Card [Blocked #773]

### ISSUE THREAD

**Issue by rt4914** [oppia/oppia-android#343]
`ReviewCardActivity` needs to be pixel perfect with the mocks.
Portrait Mock:  https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/screen/5d907b23-a75e-4f8b-8c59-afd3c734c53e/RM-Revision-Mode-
Landscape Mock is currently not available but can be done with the help of portrait mode.

Mocks have been created for HDPI Devices.
Make sure that the screen passes the accessibility test also.
For reference you can checkout: https://github.com/oppia/oppia-android/wiki/Working-on-UI

**Comment by rt4914** [comment]
Assigned to @thelosergeek

**Comment by iaminamkhan** [comment]
I think you're talking about this card activity:

https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/screen/1adacef0-f245-4115-a0e0-d1c2e7d5a0bb/TP-Revision-

Because in the above link I can see that's a text view only.

Please correct me if I am wrong.

**Comment by rt4914** [comment]
@thelosergeek Actually I the link is correct but I see that the current app is missing some data. So I will de-assign you from this issue and assign you a new issue.

**Comment by iaminamkhan** [comment]
@rt4914 I think the app is still missing some data.
Can you please check it?

**Comment by rt4914** [comment]
@thelosergeek This will take time in fixing and that's why I have de-assigned you from this. I will assign you new PR once your previous PR is finished. Also, you have created a wrong PR. I have mentioned details on your PR. https://github.com/rt4914/oppia-android/pull/1
